tools/ci/checkout-manifest-schema-branch.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+set -euxo pipefail
+
+cd "$(dirname "$0")"
+
+schema_version="$(cargo metadata --format-version=1 --no-deps | jq -r '.packages[] | select(.name == "install-action-manifest-schema") | .version')"
+branch="manifest-schema-${schema_version}"
```

Comment:
But branch name here is `manifest-schema-<major>.<minor>.<patch>`.

Reply:
I think we should use `manifest-schema-<major>.<minor>` until manifest-schema crate become 1.x, as I said in https://github.com/taiki-e/install-action/pull/657#issuecomment-2386348640.

Reply:
Thanks updated